repository/channel_impl.go

```diff
@@ -439,6 +439,57 @@ func (repo *GormRepository) GetChannelStats(channelID uuid.UUID) (*ChannelStats,
 	}
 
 	var stats ChannelStats
+	if err := repo.db.Unscoped().
+		Model(&model.Message{}).
+		Select("COUNT(channel_id) As total_message_count").
```

Comment:
細かいですが`As`や`join`のような、テーブル名/カラム名でないキーワードを大文字に（`AS`, `JOIN`）揃えてくれると見やすくて嬉しいです
```suggestion
		Select("COUNT(channel_id) AS total_message_count").
```